### PR TITLE
Protection card

### PR DESCRIPTION
**Pull request by weberjavi**
### Description
This PR updates the Protection card on the local scene sidebar. 
image

The [merged WDPA layer](https://eowilson.maps.arcgis.com/home/item.html?id=0083c9adc3b548e78a9dfbaa4c909812) has been added for testing purposes on the scene settings, on the next iteration of this task this layer should be only activated on the local scene.

Also the African country flags have been added to test how the flags could be dinamically added depending on the selected country ISO code. Once this approach is approved the rest of the flags should be provided by the design team (there's already an ongoing [task for this purpose](https://half-earth-map.atlassian.net/browse/HE-32))
The flags files have been added to the `public` folder, find documentation about it on the [React docs](https://create-react-app.dev/docs/using-the-public-folder/). 
### Designs
The updated styles could be found on [the InVision prototype](https://projects.invisionapp.com/d/main#/console/17484384/421887229/preview)
### Testing instructions
From the data globe select a country and check the sidebar (in order to see some flag visit an African country)
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-48
https://half-earth-map.atlassian.net/browse/HE-41
https://half-earth-map.atlassian.net/browse/HE-27

### CODE REVIEW

**Comment by kfilip94** [comment]
- Change font-color and font-family here:
image
image

- Increase the left/right padding:
Screenshot from 2020-07-28 12-16-03

**Comment by kfilip94** [comment]
We need a placeholder/to hide the SVG flag for countries which don't have one provided:
image

File: src/components/local-scene-sidebar/local-priority-card/local-priority-card-component.jsx [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
-              Half-Earth goal.
+            The brightly colored map layer presents one possible vision 
+            of the areas needed to achieve the Half-Earth goal of comprehensive
+            terrestrial biodiversity conservation. Higher values indicate
+            locations within the country that contribute more to the 
+            conservation of species habitat.
             </p>
             <p className={styles.datasetSource}>
-              Source: Map Of Life (Yale University)
+              Source: Source: Rinnan DS and Jetz W, (2020).
```
Comment (kfilip94) [inline src/components/local-scene-sidebar/local-priority-card/local-priority-card-component.jsx:41]:
```suggestion
              Source: Rinnan DS and Jetz W, (2020).
```

**Comment by kfilip94** [comment]
Hey @weberjavi, good job with this! Check my comments above :point_up:

**Comment by weberjavi** [comment]
> We need a placeholder/to hide the SVG flag for countries which don't have one provided:
> image

@fannycc can you provide a placeholder image to avoid displaying the broken image on the countries (places) without a flag on this first iteration?

**Comment by weberjavi** [comment]
@kfilip94 :point_up: I will leave the placeholder flag implementation for the moment when we add the remaining flags to the rest of the countries (this can be tracked [in this Jira ticket](https://half-earth-map.atlassian.net/browse/HE-32))

**Comment by weberjavi** [comment]
@kfilip94 I've commited your suggestion.
Regarding the spaces updates I've been talking to @fannycc and we've decided to keep the paddings consistent with the `current protection` part of the card.
Also regarding font sizes we will make an assessment of the state of the platform in case some other styles need to be added to the `typography.extends.scss` and the UI Kit. [This Jira task](https://half-earth-map.atlassian.net/browse/HE-61) has been created to track this issue.

**Comment by weberjavi** [comment]
@kfilip94 if you don't mind gonna merge this in order to deploy to the demo site. Is that ok?

**Pull request merged**